tests/refer.spec.ts

```diff
@@ -55,6 +57,14 @@ const showsPersonalDetails = async (page: Page): Promise<void> => {
   await page.getByRole('link', { name: 'Back', exact: true }).click()
 }
 
+const entersAReason = async (page: Page): Promise<void> => {
+  await page.getByRole('link', { name: 'Add reason for referral and any additional information' }).click()
+  await expect(page.locator('h1')).toHaveText('Add reason for referral and supporting information')
+  await page.getByLabel('Add reason for referral and supporting information').fill('This is a test reason.')
```

Comment:
We discussed in the meeting just now which version of the designs we should be referring to. Chanelle said should be using the WIP one at the bottom (=== V4), not V3, so the heading/label here will need updating to `Add reason for referral and any additional information`. Happy to leave that for a separate pair of PRs though